server/plugin/template.go

```diff
@@ -314,7 +314,6 @@ Assignees: {{range $i, $el := .Assignees -}} {{- if $i}}, {{end}}{{template "use
 	template.Must(masterTemplate.New("newReviewComment").Funcs(funcMap).Parse(`
 {{template "repo" .GetRepo}} New review comment by {{template "user" .GetSender}} on {{template "pullRequest" .GetPullRequest}}:
 
-{{.GetComment.GetDiffHunk}}
```

Comment:
Can you provide a screenshot for what this looks like?

Reply:
**Old UI**
![Screenshot from 2024-06-21 12-46-20](https://github.com/mattermost/mattermost-plugin-github/assets/90389917/d9077533-2a56-4ce6-9c9f-64196b6cb29b)

**New UI**
![new](https://github.com/mattermost/mattermost-plugin-github/assets/90389917/8c5ebbdc-0443-48ff-b2f5-ee32041f6076)